app/routes/participants/profile.js

```diff
@@ -4,9 +4,9 @@ export default Ember.Route.extend({
     model(params) {
         return Ember.RSVP.hash(
             {
-                'profile': this.store.findRecord('profile', params.profile_id),
+                'profile': this.store.query('account', {'profiles.profileId': params.profile_id}),   // TODO: Requires globally unique profile IDs- format <acctShortId>.profileId
```

Comment:
Not sure how I feel about this. What if we enforce all `profileId`s are of the format: 
`<account_id>-<profile_id>`

This would dramatically reduce the likelihood of collision by ensuring a unique prefix for each set of profiles.